### ISSUE TITLE
[🐛] Prepare "manipulateClock" test in basket.spec.ts for Cypress 12.x

### ISSUE THREAD

**Issue by bkimminich** [juice-shop/juice-shop#2087]
Up to Cypress 11.x the test works fine, but with 12.x it no longer manages to solve the challenge:

<img width="988" alt="image" src="https://github.com/juice-shop/juice-shop/assets/3531020/0883d5da-bb8a-44dc-97af-e3e10199f19d">

**Comment by ManavMalhotra** [comment]
Dear @bkimminich,

I am currently in the process of setting up the project locally, but I have encountered an error that I am unable to resolve. The error is as follows:

![image](https://github.com/juice-shop/juice-shop/assets/64147782/d6daa628-71b7-47f7-b116-22c11373782e)

I am attempting to set up the project on my Windows Subsystem for Linux (Ubuntu). Despite spending six hours trying to troubleshoot this issue, I have not been able to find a working solution.

I would greatly appreciate any assistance you could provide in resolving this issue.

Thank you for your time and consideration.

**Comment by bkimminich** [comment]
That error message is from running which command? `npm test`? Does the application work when you install and start it?

**Comment by ManavMalhotra** [comment]
Apologies for the delayed response. I've been attempting to set up on my WSL, but unfortunately, I was unsuccessful. However, I managed to get it running on my Windows machine using `npm start`, and it's working fine now.

However, I encountered an error when running `npm test`. I've been trying to resolve this issue on my own but haven't been successful yet.

Error: 
![image](https://github.com/juice-shop/juice-shop/assets/64147782/571fe9f2-1348-43b0-adee-7069123df909)
